packages/database/package.json

```diff
@@ -13,10 +13,10 @@
     "db:create-saml-database:deploy": "env SAML_DATABASE_URL=\"${SAML_DATABASE_URL}\" tsx ./src/scripts/create-saml-database.ts",
     "db:create-saml-database:dev": "dotenv -e ../../.env -- tsx ./src/scripts/create-saml-database.ts",
     "db:push": "prisma db push --accept-data-loss",
-    "db:up": "docker compose up -d",
+    "db:up": "docker compose -f ../../docker-compose.dev.yml up -d",
```

Comment:
can we remove the commands from the database package and instead add them to the root `package.json` because this is necessary for the general dev environment and not specific to the postgres database?